src/machine/machinectl.c

```diff
@@ -1119,20 +1121,24 @@ static int copy_files(int argc, char *argv[], void *userdata) {
                 host_path = abs_host_path;
         }
 
+        if (arg_force)
+                copy_flags |= COPY_FILE_REPLACE;
+
         r = bus_message_new_method_call(
                         bus,
                         &m,
                         bus_machine_mgr,
-                        copy_from ? "CopyFromMachine" : "CopyToMachine");
+                        copy_from ? "CopyFromMachineWithFlags" : "CopyToMachineWithFlags");
```

Comment:
You might have missed the comment but Lennart mentioned it's actually good to keep using the old methods if --force is not used, because machinectl can be used to call D-Bus methods on a remote host that's running an older version of systemd-machined. Sorry for the churn. Maybe add a function like this that returns the correct method name:

```
static const char *select_copy_method(bool copy_from, bool force) {
        ...
}
```

And then after adding the common arguments, check if `arg_force` is true and if so, append one extra argument with the COPY_REPLACE flag.

```
if (arg_force) {
        r = sd_bus_message_append(m, "t", COPY_REPLACE);
        ...       
}